jbpm-console-ng-generic/jbpm-console-ng-generic-client/src/main/java/org/jbpm/console/ng/gc/client/cell/longContentCell.java

```diff
@@ -0,0 +1,54 @@
+package org.jbpm.console.ng.gc.client.cell;
+
+import com.google.gwt.cell.client.AbstractSafeHtmlCell;
+import com.google.gwt.safehtml.shared.SafeHtml;
+import com.google.gwt.safehtml.shared.SafeHtmlBuilder;
+import com.google.gwt.text.shared.SafeHtmlRenderer;
+import com.google.gwt.text.shared.SimpleSafeHtmlRenderer;
+
+public class longContentCell extends AbstractSafeHtmlCell<String>{
```

Comment:
If this is eventually merged (I see you're talking to salaboy on #uberfire) then this class should probably be renamed LongContentTextCell - "L" and "Text" to mark it's usage similar to "TextCell".